httpmw/limit_per_user.go

```diff
@@ -0,0 +1,48 @@
+package httpmw
+
+import (
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-redis/redis/v8"
+	"github.com/protsack-stephan/gin-toolkit/httperr"
+)
+
+// LimitPerUser middleware is used to limit number of request per second for user.
+func LimitPerUser(cmdable redis.Cmdable, limit int) gin.HandlerFunc {
+	redisLimiter := NewRedisLimiter(cmdable, limit, time.Second*1)
```

Comment:
any reason why it's outside of callback at line 15 ?

Reply:
I think we would like 1 instance of limiter to minimize amount of redis clients.